### ISSUE TITLE
“Very” is very restrictive

### ISSUE THREAD

**Issue by mcepl** [amperser/proselint#1308]
I love that advice by Mark Twain about “very” as every other guy, but proselint is just doing too much:
```rst
“Yes, Harry, how are you on this fine evening?”

“Very well, Headmaster, thank you. It is so lovely day, isn’t it?”
```
Sentence starting with "Very well" is just too common to be highlighted as wrong.

**Comment by mcepl** [comment]
I am actually a bit ambiguous on this my own ticket: yes, on the one hand there is high number of false positives, but I am ashamed to admit how many of them are not and `proselint` forced me many times to improve my text. Perhaps, we should all suffer all those false positives for the good it brings otherwise?

**Comment by Nytelife26** [comment]
I'm glad to hear that proselint has helped you to improve your writing, @mcepl! That's great news. The design philosophy of proselint is that saying the wrong thing is worse than saying nothing, though. Out of interest, do you still feel that flagging "very well" constitutes a false-positive?

**Comment by mcepl** [comment]
Could we have some whitelist of exceptions from a rule?
